root/etc/e-smith/events/actions/nethserver-ejabberd-restore

```diff
@@ -22,17 +22,14 @@
 
 # Test if the service is running before to restore
 /usr/bin/systemctl is-active --quiet ejabberd
-
-if [[ $? -eq 0 ]]; then
-    if [ -f /var/lib/nethserver/ejabberd/ejabberd.backup ]; then
-        /opt/ejabberd-*/bin/ejabberdctl install_fallback /var/lib/nethserver/ejabberd/ejabberd.backup >/dev/null
-    else
-        echo 'The backup file is not available, restore is not possible'
-        exit 0
-    fi
+if [[ $? != 0 ]]; then
+    echo '[WARNING] Ejabberd is not running, restore is not possible'
+    exit 0
```

Comment:
Isn't that an ERROR condition that should make the restore event fail?

Reply:
yes if the restore could fail if ejabberd is not running